index.js

```diff
@@ -14,9 +14,18 @@
 //. [![CircleCI](https://img.shields.io/circleci/project/github/sanctuary-js/sanctuary/master.svg)](https://circleci.com/gh/sanctuary-js/sanctuary/tree/master)
 //. [![Gitter](https://img.shields.io/gitter/room/badges/shields.svg)](https://gitter.im/sanctuary-js/sanctuary)
 //.
-//. Sanctuary is a functional programming library inspired by Haskell
-//. and PureScript. It's stricter and more opinionated than [Ramda][],
-//. and provides a similar suite of functions.
+//. Sanctuary is a JavaScript functional programming library inspired by
+//. Haskell and PureScript. It's stricter than [Ramda][], and provides a
+//. similar suite of functions.
+//.
+//. Sanctuary promotes programs composed of simple, pure functions.
+//. Such programs are easy to test, comprehend, and maintain, and a
```

Comment:
@Jacse, since this pull request was your initiative and all I have done is apply patches based on suggestions (including a few of my own suggestions), I will squash the patches into your original commit. You will be marked as the author, I will be marked as the committer, and we will have a clean commit history. :)